source/extensions/omni.isaac.lab/omni/isaac/lab/sensors/ray_caster/patterns/patterns_cfg.py

```diff
@@ -96,6 +96,53 @@ class PinholeCameraPatternCfg(PatternBaseCfg):
     height: int = MISSING
     """Height of the image (in pixels)."""
 
+    @classmethod
+    def from_intrinsic_matrix(
+        cls,
+        focal_length: float,
+        intrinsic_matrix: list[float],
+        width: int,
+        height: int,
+    ) -> PinholeCameraPatternCfg:
+        """Create a PinholeCameraPatternCfg from an intrinsic matrix. The intrinsic matrix is a 3x3 matrix that
+        defines the mapping between the 3D world coordinates and the 2D image. The matrix is defined as:
+
+        .. math::
+            \\begin{bmatrix}
+            f_x & 0 & c_x \\\\
+            0 & f_y & c_y \\\\
+            0 & 0 & 1
+            \\end{bmatrix}
+
+        Args:
+            focal_length: Focal length of the camera (in cm).
+            intrinsic_matrix: Intrinsic matrix of the camera (shape: (9,)).
```

Comment:
Somehow colons aren't liked by doc generator.

```suggestion
            intrinsic_matrix: Intrinsic matrix of the camera in row-major format. 
 Shape is (9,).
```